src/one_dragon_qt/overlay/overlay_manager.py

```
@@ -711,128 +613,79 @@ def _vision_item_iou(a, b) -> float:
-                ("NodeRetry", retry_times),
-            ]
-        )
-
-        if hasattr(self.ctx, "auto_battle_context"):
-            items.extend(self._collect_auto_battle_items())
-
-        return items
-
-    def _collect_auto_battle_items(self) -> list[tuple[str, str]]:
+    def _build_state_title(self) -> str:
+        """状态面板标题：当前前台角色名（如「前台-艾莲」），无则回退「战斗状态」。"""
```

Ruff (0.16.2)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)